pywt/_extensions/c/wavelets.c

```diff
@@ -316,123 +349,342 @@ Wavelet* wavelet(char name, unsigned int order)
                 }
             }
             break;
+        default:
+            return NULL;
+    }
+    return w;
+}
+
+ContinuousWavelet* continous_wavelet(WAVELET_NAME name, unsigned int order)
+{
+    ContinuousWavelet *w;
+    switch(name){
+            /* Gaussian Wavelets */
+        case GAUS:
+            if (order > 8)
+                return NULL;
+            w = blank_continous_wavelet();
+            if(w == NULL) return NULL;
 
+            w->base.vanishing_moments_psi = -1;
```

Comment:
These attributes should probably live on the discrete wavelet struct, since they are not define for any continuous wavelet. Thoughts?